README.md

```diff
@@ -15,136 +15,71 @@ When you have set up your repo
-├── miss_hit.cfg        # configuration file for the matlab miss hit linter
-└── initEnv.m           # a .m file to set up your project (adds the right folder to the path)
-```
-
-## Keeping your code stylish: miss hit linter
+- anatomical defacing [ W I P ]
 
-### Matlab code style guide and quality
+- gzip the nii files in a BIDS raw folder
 
-We use the [MISS_HIT linter](https://github.com/florianschanda/miss_hit/) to
-automatically enforce / fix some code style issues and check for code quality.
+The core functions are in the `src` folder. This set of functions is intended to be added manually
+as a submodule to the analysis repo forked from
+[template_fMRI_analysis](https://github.com/cpp-lln-lab/template_fMRI_analysis)
 
-The linter is a python package that can be installed with:
+## How to install and use this template
 
-```bash
-pip3 install --user miss_hit
-```
+### Add as a submodule
 
-The rules followed by MISS_HIT are in the
-[MISS_HIT configuration file](./miss_hit.cfg).
+1. Open the terminal and navigate to your analysis folder forked from
+[template_fMRI_analysis](https://github.com/cpp-lln-lab/template_fMRI_analysis) and cloned locally
 
-To check the code style of the whole repository, you can can simply type:
+2. Type
 
 ```bash
-mh_style .
-```
+# navigate to the `lib` folder
+cd lib
 
-Some styling issues can be automatically fixed by using the `--fix` flag. You
-might need to rerun this command several times if there are a lot of issues.
+# add the submodule
+git submodule add https://github.com/cpp-lln-lab/CPP_sourceMRI
 
-```bash
-mh_style . --fix
+# initialize the nested submodule(s)
+git submodule update --init --recursive
 ```
 
-Code quality can be checked with:
+3. commit the new changes (the new submodule(s))
 
-```bash
-mh_metric .
-```
+### Dependencies
 
-To see only the issues that "break" the code quality rules set in the
-configuration file, type:
+Make sure that the following toolboxes are installed and added to the matlab
+path.
 
-```bash
-mh_metric . --ci
-```
+For instructions see the following links:
 
-The code style and quality is also checked during the
-[continuous integration](.github/workflows/miss_hit.yml).
+<!-- lint disable -->
 
-For more information about MISS_HIT see its
-[documentation](https://florianschanda.github.io/miss_hit/).
+| Dependencies                                                                              | Used version |
+| ----------------------------------------------------------------------------------------- | ------------ |
+| [Matlab](https://www.mathworks.com/products/matlab.html)                                  | 20???        |
+| or [octave](https://www.gnu.org/software/octave/)                                         | 4.?          |
+| [SPM12](https://www.fil.ion.ucl.ac.uk/spm/software/spm12/)                                | v7487        |
+| [CPP_SPM](https://github.com/sergivalverde/nifti_tools)*                                  | > 0.1.0      |
```

Comment:
the link associated to CPP_SPM seems wrong, no?

Reply:
ops! good catch
```suggestion
| [CPP_SPM](https://github.com/cpp-lln-lab/CPP_SPM)*                                        | > 0.1.0      |
```